src/Auth/LogIn/Login.module.css

```diff
@@ -1,12 +1,42 @@
-.container {
-    margin: 25vh 25vw;
+.pageContainer {
+    background-image: url("background.png");
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+}
+
+.pageContainer > h1 {
+    margin-top: 2rem;
+    font-size: 4rem;
+}
+
+.pageContainer > .header {
+    text-transform: uppercase;
+    color: white;
+}
+
+.pageContainer > p, h1 {
+    width: 100%;
+    text-align: center;
+    color: white;
+}
+
+.pageContainer a {
+    color: white;
+}
+
```

Comment:
I think these can be deleted

Reply:
All can be removed except for the link styling, this needs to be white as well.